nestkernel/event_delivery_manager.cpp

```diff
@@ -312,6 +317,9 @@ EventDeliveryManager::gather_spike_data_( const thread tid,
 
   const AssignedRanks assigned_ranks = kernel().vp_manager.get_assigned_ranks( tid );
 
+  // Assume a single gather round
+  decrease_buffer_size_spike_data_ = true;
```

Comment:
sorry to be so late to the party, but would it be possible to turn this member variable into a variable local to this function?